profile/README.md

```diff
@@ -0,0 +1,28 @@
+
+![Alternative Text](./img/openmbee-logo.svg)
+
+## Welcome to OpenMBEE
+OpenMBEE (Open Model Based Engineering Environment) is an open source collaborative engineering system. It enables engineers to work in the language of their choice and easily share and document their work across other tools.
+
+## OpenMBEE Architectures
+### Execubot
+The Execubot architecture allows Systems Engineers to develop models in their enterprise model development platform and publish aspects of the model to the web.  
+
+Popular Repositories:
+- [Model Management System](https://github.com/Open-MBEE/mms) - Management and version control for SysML models
+- [Cameo Model Development Kit](https://github.com/Open-MBEE/mdk) - Cameo Systems Modeler plugin for defining views for View Editor
+- [View Editor](https://github.com/Open-MBEE/ve) - Interact with SysML models within a web-based environment
+
+### Flexo
+Flexo expands on the concept of version control models from execubot. Execubots representation of the data is in JSON, Flexo exposes models as RDF. Models in Flexo are not limited to enterprise model development platforms. A model in Flexo can be anything that is represented as RDF.
+
+Popular Repositories:  
```

Comment:
I think we can remove this and include these in the _**pinned repositories**_...

Reply:
How about doing both?

Reply:
given that we plan to rename most of these repos, i think we can finalize this at the openmbee meeting, then there's some pattern we can give for identifying which repo belongs to which architecture